include/core/federated/network/net_common.h

```diff
@@ -670,6 +670,18 @@ THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 #define MSG_TYPE_FAILED 25
 
+/**
+ * Byte identifying a downstream next event tag (DNET) message sent
+ * from the RTI in centralized coordination.
+ * The next eight bytes will be the timestamp.
+ * The next four bytes will be the microstep.
+ * This signal from the RTI tells the destination federate the latest tag that
+ * the federate can safely skip sending a next event tag (NET) signal.
+ * In other words, the federate doesn't have to send NET signals with tags
+ * earlier than or equal to this tag unless it cannot advance to its next event tag.
```

Comment:
I think this could be clearer:
```suggestion
 * This signal from the RTI tells the destination federate that downstream federates do not need
 * for it to send any next event tag (NET) signal with a tag _g_ less than the specified tag. Thus, it should only send
 * those signals if needs permission from the RTI to advance to _g_.
```

Reply:
I applied this clearer and fancy description in [896e152](https://github.com/lf-lang/reactor-c/pull/349/commits/896e15293bd21958a124d7f73b02c455340f81d7).